core/cluster.hxx

```diff
@@ -361,9 +361,13 @@ class cluster : public std::enable_shared_from_this<cluster>
                     return handler(ec);
                 }
             } else {
-                // add the cappela Root CA if no other CA was specified.
-                std::error_code ec{};
                 LOG_DEBUG(R"([{}]: use default CA for TLS verify)", id_);
+                std::error_code ec{};
+                tls_.set_default_verify_paths(ec);
+                if (ec) {
+                    LOG_WARNING(R"([{}]: failed to load system CAs: {})", id_, ec.message());
+                }
+                // add the capella Root CA if no other CA was specified.
```

Comment:
I think this comment is now incorrect, isn't it?  It adds the capella CA in addition to adding the platform certs.

Reply:
yes, you right I will adjust the comment. Thanks for the note